packages/utils/vite.config.integration.ts

```diff
@@ -4,21 +4,14 @@ import { defineConfig } from 'vite';
 
 export default defineConfig({
   cacheDir: '../../node_modules/.vite/cli',
-
   plugins: [nxViteTsPaths()],
-
-  // Uncomment this if you are using workers.
-  // worker: {
-  //  plugins: [ nxViteTsPaths() ],
-  // },
-
   test: {
     globals: true,
     cache: {
       dir: '../../node_modules/.vitest',
     },
     environment: 'node',
     include: ['src/**/*.integration.test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
-    globalSetup: ['global-setup.e2e.ts'],
```

Comment:
Is the file 'global-setup.e2e.ts' still needed?

Reply:
This setup is used for all E2E tests and it sets up the local registry and runs the npm publish. While I do not enjoy waiting for E2E tests, I thought that this was desired.